util/linux_install.sh

```diff
@@ -149,6 +149,29 @@ elif grep ID /etc/os-release | grep -q slackware; then
 		echo "Quitting..."
 	fi
 
+elif grep ID /etc/os-release | grep -q void; then
+	# musl Void systems don't have glibc cross compilers avaliable in their repos.
+	# glibc Void systems do have musl cross compilers though, for some reason.
+	# So, default to musl, and switch to glibc if it is installed.
+	CROSS_ARM=cross-arm-linux-musleabi
+	if xbps-query glibc > /dev/null; then # Check is glibc if installed
+		CROSS_ARM=cross-arm-linux-gnueabi
+	fi
+
+	sudo xbps-install \
```

Comment:
Does `xbps-install` ask for input (Y/N) by default?

Reply:
Yep, it does. Noticed all the other ones still asked for y/n input, so I left it there.

Reply:
Could you add a comment for how to ... well, not ask for y/n input. 

I think in the future, we'd like to make this more automated, so it would be nice to have the reference, just in case.